js/kaomoji-collection.js

```diff
@@ -2,18 +2,36 @@
 
 const kaomojiCollection = {
     "kaomoji": {
-        "sparkles": [
-            "✧･ﾟ: *✧･ﾟ:* 　　 *:･ﾟ✧*:･ﾟ✧",
-            "*＊✿❀　❀✿＊*"
+        "angry": [
```

Comment:
I'd recommend to tag this list as `angry middle finger fuck` since all of them are pointing the finger